pkg/demoinfocs/common/common_test.go

```diff
@@ -213,13 +213,14 @@ type demoInfoProviderMock struct {
 	tickRate             float64
 	ingameTick           int
 	playersByHandle      map[int]*Player
+	entitiesByHandle     map[uint64]st.Entity
```

Comment:
was this supposed to be part of this MR? seems to be unrelated

Reply:
it's required for the test - the pawn entity must exist to pass the check `!p.IsAlive()` in `Velocity` (done through `FindEntityByHandle`)

Reply:
right, totally missed that :+1: